rustls/src/lib.rs

```diff
@@ -389,7 +389,7 @@ mod conn;
 pub mod crypto;
 mod error;
 mod hash_hs;
-#[cfg(feature = "std")]
+#[cfg(any(feature = "std", feature = "hashbrown"))]
```

Comment:
I think this change logically belongs with the previous one?

Reply:
Done.